test/evergreen.yml

```diff
@@ -1861,6 +1920,89 @@ tasks:
           test_config_filename: configs/reverse_split_default.txt
           test_name: reverse_split
 
+  # "cppsuite test" with ${test_config} test_config: debug_mode=(cursor_copy=true)
+  - name: cppsuite-combined-cursor-copy
```

Comment:
I'm not sure if this is what @jiechenbo had in mind when he wanted the tasks combined. @jiechenbo what do you think?

Reply:
Probably it is. He told me he wanted only the "pull_request" tagged cppsuite tasks combined. With merging cppsuite-combined-blank into cppsuite-combined-cursor-copy there are only two "pull_request" tagged cppsuite tasks left:  cppsuite-background-compact-default and cppsuite-combined-cursor-copy. These have different test_config values.

Never-the-less, I wait for Jie's response.

Reply:
The change follows in the direction of what i was thinking. I think we can instead create a shell script inside evergreen to run all the tests. With regarding the WT_TEST being overridden, I think it is fine as long as we stop when the cppsuite test fails, which the cppsuite test suite already doess when running all the tests.